src/compas/geometry/polyhedron.py

```diff
@@ -270,7 +278,7 @@ def __or__(self, other):
 
     @property
     def vertices(self):
-        if not self._vertices:
+        if self._vertices is None:
```

Comment:
In this case, I think the falsey-ness check (`if not self._vertices`) is a better option

Reply:
This caused an error message on a quick test. The error message is 
```
The truth value of an array with more than one element is ambiguous.
```